src/components/MainPage/HotPrices/HotPrices.scss

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+main,
+h1,
+h2,
+h3,
+h4,
+h5,
+p,
+section,
+article {
+  margin: 0;
+}
+
+button {
+  padding: 0;
+}
```

Comment:
Ok for the next commits we will create reset.css file and just import in required places